code/datums/components/payment.dm

```diff
@@ -98,12 +99,32 @@
 
 		if(armless)
 			if(!user.pulling || !iscash(user.pulling) && !istype(user.pulling, /obj/item/card/id))
-				to_chat(user, span_notice("Try pulling a valid ID, space cash, holochip or coin while using \the [parent]!"))
+				//to_chat(user, span_notice("Try pulling a valid ID, space cash, holochip or coin while using \the [parent]!"))
+				switch(transaction_style)
```

Comment:
I think tg might appreciate this change but if you don't want to PR there I understand that.
I wonder if a better fix might be instead to make `transaction_style` a datum with three children which just holds appropriate strings and instantiate the string collection instead of doing a bunch of case statements but that is out of scope here and would be a maintainance problem if you didn't push it upstream.

Reply:
I didn't think about the maintenance problem, but I should have.  I'll put it upstream at some point.

Reply:
Oh, I don't think it will be a maintenance problem as-is, this isn't very often touched code.
I meant that if you made a more substantial refactor then it might be 😅